src/applications/static-pages/covid-vaccine-updates-cta/On.jsx

```diff
@@ -11,48 +11,38 @@ import {
 
 import { rootUrl as covidVaccineFormUrl } from 'applications/coronavirus-vaccination/manifest.json';
 
-const buttonText = 'Sign up to stay informed';
-
-function renderDowntime(downtime, children) {
-  if (downtime.status === 'down') {
-    return <button disabled>{buttonText}</button>;
-  }
-  return children; // Render normal enabled button
-}
-
-function OnState() {
+function OnState({ copy }) {
   return (
     <>
       <DowntimeNotification
         dependencies={[externalServices.vetextVaccine]}
-        appTitle="Covid 19 Vaccination Information"
+        appTitle={copy.appTitle}
       >
         <div />
       </DowntimeNotification>
       <AlertBox
         status={ALERT_TYPE.INFO}
-        headline="Stay informed about getting a COVID-19 vaccine"
+        headline={copy.headline}
         content={
           <>
+            <p>{copy.cta}</p>
+            <p>{copy.body}</p>
             <p>
-              Sign up for an easy way to stay informed about getting a COVID-19
-              vaccine at VA.
-            </p>
-            <p>
-              When you sign up, we’ll also ask about your vaccine plans. Your
-              local VA health facility may use this information to determine
-              when to contact you once your risk group becomes eligible.
-            </p>
-            <p>
-              <strong>Note:</strong> We’ll contact every eligible Veteran in
-              each risk group. You don’t need to sign up to get a vaccine.
+              {' '}
+              {copy.boldedNote ? <strong>{copy.boldedNote} </strong> : ''}
+              {copy.note}
```

Comment:
I think I would have just made an entirely separate Spanish version of this component

```jsx
function OnState({ language }) {
    if (language == SPANISH) {
        return <SpanishWidget/>
    }.

    return <EnglishWidget/>
}
```

It would have some repetitiveness between components but this way keeps it more flexible in case the content structure starts to diverge between English and Spanish.

Reply:
got it... for now (and since we also have to handle tagalog) I am going to re use the same component and I can create different versions post launch, if needed